package.json

```
@@ -52,10 +59,13 @@
     "@babel/plugin-transform-flow-strip-types": "7.10.4",
     "@babel/preset-env": "7.10.4",
     "@babel/register": "7.10.4",
+    "@types/accepts": "^1.3.5",
```

Please use `-E` flag for `npm install`
```suggestion
    "@types/accepts": "1.3.5",
```

---

Thanks sorry keep forgetting ☹️